invenio_files_rest/utils.py

```diff
@@ -35,3 +35,12 @@ def load_or_import_from_config(key, app=None, default=None):
     app = app or current_app
     imp = app.config.get(key)
     return obj_or_import_string(imp, default=default)
+
+
+def get_extra_encodings(encoding):
```

Comment:
I would extra the entire function instead of only part of it.

```python
def guess_mimetype(filename):
    m, encoding = mimetypes.guess_type(filename)
    if encoding:
        m = ENCODING_MIMETYPES.get(encoding, None)
    return m or 'application/octet-stream'
```